tracer/src/Datadog.Trace/ClrProfiler/ServerlessInstrumentation/ServerlessMiniAgent.cs

```diff
@@ -0,0 +1,113 @@
+// <copyright file="ServerlessMiniAgent.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.IO;
+
+using Datadog.Trace.Logging;
+
+namespace Datadog.Trace.ClrProfiler.ServerlessInstrumentation;
+
+internal class ServerlessMiniAgent
```

Comment:
This seems to be a helper class with no state, so you can make it `static`:
```suggestion
internal static class ServerlessMiniAgent
```

Reply:
fixed